app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,111 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* Variables */
+// COLORS
+// ======
+$light-gray: #777;
+$mega-light-gray: #eaeaea;
+$dark-gray: #555;
+$near-black: #222;
+$white: #fff;
+
+/* General Config */
+body {
+  padding-top: 60px;
+}
+
+section {
+  overflow: auto;
+}
+
+textarea {
+  resize: vertical;
+}
+
+.center {
+  text-align: center;
+
+  h1 {
+    margin-bottom: 10px;
+  }
+}
+
+/* Typography */
+h1, h2, h3, h4, h5, h6 {
+  line-height: 1;
+}
+
+h1 {
+  font-size: 3em;
+  letter-spacing: -2px;
+  margin-bottom: 30px;
+  text-align: center;
+}
+
+h2 {
+  font-size: 1.2em;
```

Comment:
Properties should be ordered color, font-size, font-weight, letter-spacing, margin-bottom, text-align